docs/running-locally.md

```diff
@@ -0,0 +1,22 @@
+# Running Epidisco With Ketrew/Coclobas Locally
```

Comment:
I wouldn't call that "Running locally" as it could be interpreted as "run the whole pipeline and everything on your laptop"?

This is more about "submitting to Ketrew from another host, esp. your local machine"